frontend/tests/cypress/component/ccofApplication/CcofApplicationTypeSelector.cy.js

```diff
@@ -0,0 +1,91 @@
+import CcofApplicationTypeSelector from '@/components/ccofApplication/CcofApplicationTypeSelector.vue';
```

Comment:
There's an additional component NewAppIntermediatePage that comes before the Family/Group selector.